### ISSUE TITLE
Incomplete description

### ISSUE THREAD

**Issue by andre-simon** [bezhermoso/base16-highlight#1]
Hi,

in  base16-highlight/templates/default.mustache it should read 

`Description = "Base16 {{scheme-name}}"  `

Offtopic: What about providing a release which can be linked as additional "theme pack" from the highlight site?

**Comment by tajmone** [comment]
I agree with André. I was working on some custom theme resources to add to Highlight's "extras" folder when I came across this, and I'd like to reuse it.

Please, notice that there is an error in the mustache template, at line 16:

- https://github.com/bezhermoso/base16-highlight/blob/6fcc72e/templates/default.mustache#L16

where `Colour` is mispelled as `Color`:

```lua
LineNum      = { Color="#{{base03-hex}}" }
```

... this error has been propagated in all the built themes, so it would need fixing and rebuilding all themes.

**Comment by bezhermoso** [comment]
@andre-simon I went ahead and generated a GitHub release archives w/ `highlight-themes/*.theme` files for all Base16 schemes.

https://github.com/bezhermoso/base16-highlight/releases/tag/2023.07.22

How would you like these & further updates be available as "theme packs" on your website?
